components/video.js

```diff
@@ -1,50 +0,0 @@
-import { useEffect, useRef, useCallback } from 'react'
-import { useInView } from 'react-intersection-observer'
-import 'intersection-observer'
-
-const Video = ({ src, ...props }) => {
-  const [inViewRef, inView] = useInView({
-    threshold: 1
-  })
-  const videoRef = useRef()
-
-  const setRefs = useCallback(
-    node => {
-      // Ref's from useRef needs to have the node assigned to `current`
-      videoRef.current = node
-      // Callback refs, like the one from `useInView`, is a function that takes the node as an argument
-      inViewRef(node)
-
-      if (node) {
-        node.addEventListener('click', function() {
-          if (this.paused) {
-            this.play()
-          } else {
-            this.pause()
-          }
-        })
-      }
-    },
-    [inViewRef]
-  )
```

Comment:
Is there a way to add this logic back? So when the video is not in the viewport, it pauses automatically.

Given these videos are not short, having this feature will help a lot.

Reply:
Discussed in DM. Not a blocker and we'll merge it for now.